Opcionales y obligatorios

aldanacase opened the pull request:
Link de Vercel:

https://digital-dragons-3v4uz1xut-digitaldragons.vercel.app

GeroLeLan committed:
Actualización de next

aldanacase committed:
Update package-lock.json

aldanacase committed:
Merge branch 'Proyecto-Javascript---React/Vue' of https://github.com/iaw-2023/DigitalDragons-js into Proyecto-Javascript---React/Vue

GeroLeLan committed:
Creando la barra de navegación

GeroLeLan committed:
Vista de barra de navegacion y pagina principal

GeroLeLan committed:
Creando el formulario para buscar vuelos por ciudades

GeroLeLan committed:
Ahora la home utiliza la api para obtener las ciudades

GeroLeLan committed:
Creando el componete para mostrar los vuelos

GeroLeLan committed:
Empeizo a obtener los vuelos mediante la API

Por ahora sigue usando un par arreglo hardcodeado como vuelos de ejemplo

aldanacase committed:
Update Navbar.js

Para pestaña reservas

GeroLeLan committed:
Funciona el filtro de vuelos por ciudad de orgen y destino

GeroLeLan committed:
Pagina para reservar un vuelo en particular

GeroLeLan committed:
Selección de categoría para hacer la reserva de un vuelo

GeroLeLan committed:
Ya se pueden hacer reservas

aldanacase committed:
Construyendo pagina de Reservas

Se agregó la pagina principal para las reservas junto con el componente que se utiliza para mostrar las reservas. Se generaron ejemplos de prueba.

GeroLeLan committed:
AboutUs, Footer y algunos detalles

GeroLeLan committed:
Ahora se controla la cantidad de asientos antes de realizar una reserva

GeroLeLan committed:
Cambios en el mensaje de reserva realizada

aldanacase committed:
Ahora se muestran las reservas sacando los datos de la API.

Se muestran las reservas sacando los datos de la API. Se actualizaron algunos detalles.

GeroLeLan committed:
Ahora se obliga a seleccionar una categoria

GeroLeLan committed:
Merge branch 'Proyecto-Javascript---React/Vue' of https://github.com/iaw-2023/DigitalDragons-js into Proyecto-Javascript---React/Vue

GeroLeLan committed:
Agrego paginado a la pagina de reservas

GeroLeLan committed:
Cambio el navbar

aldanacase committed:
Ultimos cambios para la pagina de reservas y detalles

aldanacase committed:
Merge branch 'Proyecto-Javascript---React/Vue' of https://github.com/iaw-2023/DigitalDragons-js into Proyecto-Javascript---React/Vue

aldanacase committed:
Update Footer.js

aldanacase committed:
Update layout.js

aldanacase committed:
Pruebas Card Payment Brick Mercado Pago

GeroLeLan committed:
Prueba de Mercado Pago

GeroLeLan committed:
Merge branch 'Opcionales-y-Obligatorios' of https://github.com/iaw-2023/DigitalDragons-js into Opcionales-y-Obligatorios

GeroLeLan committed:
pruebas de mercado pago en react

GeroLeLan committed:
Implementacion del Login

GeroLeLan committed:
Mejoras en el login

aldanacase committed:
Probando el funcionamiento de MP

aldanacase committed:
Probando MP, cambiando algunas configuraciones de esLint

aldanacase committed:
ultimas pruebas MP

GeroLeLan committed:
Login/logout funcionales

Aunque falta pulir mucho

aldanacase committed:
Funcionando la integracion de MercadoPago

GeroLeLan committed:
register

GeroLeLan committed:
Merge branch 'Opcionales-y-Obligatorios' of https://github.com/iaw-2023/DigitalDragons-js into Opcionales-y-Obligatorios

GeroLeLan committed:
actualizando la pagina de vuelos

aldanacase committed:
Algunas correcciones de MP y Prueba Mostrar reservas de un cliente (incompleto)

GeroLeLan committed:
La reserva ahora se realiza cuando pagas

GeroLeLan committed:
Merge branch 'Opcionales-y-Obligatorios' of https://github.com/iaw-2023/DigitalDragons-js into Opcionales-y-Obligatorios

GeroLeLan committed:
Intenando mostrar una foto la page de vuelo

aldanacase committed:
Algunos arreglos en aboutUs, mercadoPago y Reservas

aldanacase committed:
Mejoras en pagina Reservas y cambios generales.

aldanacase committed:
Actualizacion FlightPage para mostrar imagen

aldanacase committed:
Update page.js

dcm-uns reviewed (changes requested):
Para la promoción faltaría:

- Un servicio web en React
- Que la aplicación React sea responsive.

aldanacase committed:
Servicio Web en React: ya se ven las imagenes.

aldanacase committed:
Servicio Web en React:pruebas

aldanacase committed:
Pruebando el servicio web de react

aldanacase committed:
Pruebas con el servicio web de React

aldanacase committed:
Pruebas API externa React

aldanacase committed:
prueba en flight

aldanacase committed:
ultimos cambios servicio web react

GeroLeLan committed:
ahora es resposive lo que antes era una tabla

ademas anda el boton de reserva que se habia roto

GeroLeLan committed:
ahora el home es mas responsive

GeroLeLan committed:
Cambios menores

ya no se abre una nueva pestaña cuando le das a realizar reserva en el home y ahora el msg de cargando de reservas es igual al otro msg de cargando

aldanacase commented:
Link de Vercel: 

https://digital-dragons-joq5s33tr-digitaldragons.vercel.app

Opcionales realizados para la promoción:

-Servicio Web en React
-Aplicación React Responsive
